py/desitarget/secondary.py

```diff
@@ -102,6 +103,113 @@
 ])
 
 
+def match_to_main_survey(ras, decs, sep=1.):
+    """Standalone code to match RA/Dec positions to Main Survey targets.
+
+    Parameters
+    ----------
+    ras : :class:`~numpy.array` or `list`
+        Right Ascensions of interest (degrees).
+    decs : :class:`~numpy.array` or `list`
+        Declinations of interest (degrees).
+    sep : :class:`float`, defaults to 1 arcsecond
+        Separation (arcsec) to match `ras`/`decs` to Main Survey targets.
+
+    Returns
+    -------
+    :class:`~numpy.array`
+        An array that includes 6 columns:
+            RA: The input `ras`
+            DEC: The input `decs`
+            BRIGHT: ``True`` for locations matching a bright-time target.
+            DARK: ``True`` for locations matching a dark-time target.
+            BRIGHTSEC: ``True`` for matching a bright-time secondary.
+            DARKSEC: ``True`` for matching a dark-time secondary.
+
+    Notes
+    -----
+    - Must have set up the correct environment by running, e.g.,
+      source /global/cfs/projectdirs/desi/software/desi_environment.sh
+    - Prints summary statistics of the number of matches to screen.
```

Comment:
I think the preferred, current location of the desi software environment is `/global/common/software/desi/desi_environment.sh`.